blockchain/v1/reactor_test.go

```diff
@@ -50,7 +50,12 @@ func makeVote(
 	blockID types.BlockID,
 	valset *types.ValidatorSet,
 	privVal types.PrivValidator) *types.Vote {
-	addr := privVal.GetPubKey().Address()
+	pv, err := privVal.GetPubKey()
+	if err != nil {
+		panic(err)
```

Comment:
This is bordering on a nitpick, but for these testing helper functions, you can have them take a `*testing.T` as a param and then call `t.Fatal` instead of panicking.

Reply:
made the change. thanks